config/initializers/rack_attack.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class Rack::Attack
+  Rack::Attack.throttle('limit sitemap requests by IP', limit: 1, period: 10) do |request|
+    request.ip if request.path == '/sitemap' && request.get?
```

Comment:
Is it possible to make this path match by regex? I know the index page is the largest & slowest to load, but I think the misbehaved crawlers we've seen have been going through the individual sitemap pages (e.g. https://catalog-staging.princeton.edu/sitemap/0004)

Reply:
Done